tests/Facebook/InstantArticles/Elements/InstantArticleTest.php

```diff
@@ -284,4 +284,9 @@ public function testRenderWithAds()
 
         $this->assertEquals($expected, $this->article->render());
     }
+
+    public function testImplementsInterface()
+    {
+        $this->assertInstanceOf(InstantArticleInterface::class, $this->article);
```

Comment:
This ::class wont run on later PHP version.
This is why we moved from ::class to getClassName() method from Element base class.